src/app/notifications/components/SearchFailure/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { assertWindow, useOnDOMEvent, useTimeout } from '../../../utils';
+import { Header } from '../Card';
+import {
+  BannerBody,
+  BannerBodyWrapper,
+  clearErrorAfter,
+  CloseButton,
+  CloseIcon
+} from './styles';
+
+interface Props {
+  dismiss: () => void;
+}
+
+// tslint:disable-next-line:variable-name
+const SearchFailure = ({ dismiss }: Props) => {
+  const window = assertWindow();
+  const [isFadingOut, setIsFadingOut] = React.useState(false);
+  const [shouldAllowDismissal, setShouldAllowDismissal] = React.useState(false);
+
+  const startFadeOut = () => {
+    if (shouldAllowDismissal) {
+      setIsFadingOut(true);
+    }
+  };
+
+  useTimeout(clearErrorAfter, startFadeOut, [shouldAllowDismissal]);
```

Comment:
this is going to dismiss after startFadeOut + 100 because of the re-bind after shouldAllowDismissal changes, i think you can probably just not pass any dependencies and it'll be more clear